qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -340,9 +340,73 @@ def get_spinZ_matrix_elements(mol_name, hf_data, n_active_electrons=None, n_acti
     return spinz_matrix_elements
 
 
+def get_particle_number_table(mol_name, hf_data, n_active_electrons=None, n_active_orbitals=None):
+    r"""Reads the Hartree-Fock (HF) electronic structure data file, defines an active
+    space and generates the table required to build the particle number operator :math:`\hat{N}`.
+
+    In general, if an active space is defined, the particle number operator reads,
+
+    .. math::
+
+        \hat{N} = 2 n_{\mathrm{docc}} + \sum_\alpha \hat{c}_\alpha^\dagger \hat{c}_\alpha,
```

Comment:
When I look at this expression and then compare to the table of matrix elements, it's not easy to understand why they are all constant and equal to one. How could we explain this better?

Reply:
I have tried to clarified this in the docstring.